packages/data-broker/src/__tests__/services/dhis/pullers/EventsPuller.test.js

```diff
@@ -49,20 +38,26 @@ describe('EventsPuller', () => {
     it('correctly invokes the event analytics api in DHIS', () =>
       assertEventAnalyticsApiWasInvokedCorrectly({
         dataSources: [DATA_GROUPS.POP01_GROUP],
+        options: {
+          organisationUnitCodes: ['DL_FACILITY_A'],
+        },
```

Comment:
I guess we did technically support not passing in `organisationUnitCodes` in the options previously. I believe DhisApi throws an error in the real code if this is the case, so I don't mind introducing this requirement in the DhisService level...

Reply:
Ah cool, should we add a test to make sure it throws?

Reply:
Ah good idea, will do 👍